src/main/java/net/sf/jabref/logic/importer/fetcher/ArXiv.java

```diff
@@ -345,10 +343,9 @@ public BibEntry toBibEntry(Character keywordDelimiter) {
             abstractText.ifPresent(abstractContent -> bibEntry.setField(FieldName.ABSTRACT, abstractContent));
             getDate().ifPresent(date -> bibEntry.setField(FieldName.DATE, date));
             primaryCategory.ifPresent(category -> bibEntry.setField(FieldName.EPRINTCLASS, category));
-            journalReferenceText.ifPresent(journal -> bibEntry.setField(FieldName.JOURNALTITLE, journal));
-            getPdfUrl().ifPresent(url -> (new TypedBibEntry(bibEntry, BibDatabaseMode.BIBLATEX))
+            journalReferenceText.ifPresent(journal -> bibEntry.setField(FieldName.JOURNAL, journal));
```

Comment:
Please leave `journaltitle` here since the returned entry is in biblatex format (see date above)